modules/_canvas-kit-css/index.scss

```diff
@@ -9,7 +9,6 @@ $wdc-system-icons-path: '../../node_modules/@workday/canvas-system-icons-web/dis
 @import '~@workday/canvas-kit-css-icon/index.scss';
 @import '~@workday/canvas-kit-css-layout/index.scss';
 @import '~@workday/canvas-kit-css-loading-animation/index.scss';
-@import '~@workday/canvas-kit-css-menu/index.scss';
```

Comment:
@sahlhoff Do you mind moving the css menu back? Sorry - should have been more clear with this one. We want to keep CSS menu available regularly and just move the React menu into Labs. The styling is pretty set in stone, it's just the implementation details of the React menu that we're still figuring out.

Reply:
Ah, no worries. Probably should have checked before. 

Question on menus, https://design.workday.com/components/popups/menus, should we update the iframe to point to css if it is the correct design?